CHANGELOG.md

```diff
@@ -7,22 +7,25 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Added
+- Added `--template` option to create-nostalgie-app to allow providing a custom template
```

Comment:
Love that you did the changelog entry :)

Did you find [kacl](https://npm.im/kacl) by any chance? My workflow these days is:

```sh
# This will pop open your configured $EDITOR to prompt you for markdown. The markdown will be correctly wrapped into the changelog list.
kacl add added
```

Reply:
I did that manually, never saw kacl before tbh :)